DEPLOY.md

```diff
@@ -1,20 +1,26 @@
-## Preparation
+# RInAppMessaging iOS SDK
+
+1. [Release Workflow](#release-workflow)
+
+## Release Workflow
+
+### Preparation
 Create pull request to make the following changes:
 
-1. Update `CHANGELOG.md`
+1. Update the date in `CHANGELOG.md`
 1. Update version number in `openapi.yaml`
 1. Update version number in `.jazzy.yaml`
 1. Update version number in `RInAppMessaging.podspec`
 1. Update version number in `Sources/RInAppMessaging/Resources/Versions.plist` under `IAMCurrentModuleVersion` key
 1. Add version number to `_versions` file
 
-## Deploy
+### Deploy
```

Comment:
'Deploy' and 'Using CI' are one the same ### level ?

Reply:
Fixed.